WordPress/Classes/Utility/App Configuration/AppConstants.swift

```diff
@@ -0,0 +1,16 @@
+import Foundation
+
+struct AppConstants {
+    static let productTwitterHandle = "@WordPressiOS"
+    static let productTwitterURL = "https://twitter.com/WordPressiOS"
+    static let productBlogURL = "https://blog.wordpress.com"
+}
+
+// MARK: - Localized Strings
+extension AppConstants {
+
+    struct AppSettings {
```

Comment:
Perhaps just `Settings`? Then we can refer to it as `AppConstants.Settings.value`

Reply:
Good point, I think `AppConstants.Settings.value` works 👍

Reply:
Done 15a83a8 😄